lib/client/socket.js

```diff
@@ -42,8 +42,9 @@ module.exports = ({ url, token, filters, config }) => {
     }
   });
   io.on('open', () => {
-    logger.info({ token, url }, 'identifying');
-    io.send('identify', token);
+    logger.info({ token, url }, 'identifying with broker server');
+    const clientData = { url, token, filters, config };
+    io.send('identify', JSON.stringify(clientData));
```

Comment:
I'm stringifying the client data to parse it on the server side. Should I just send the object as is, and read it on the other side? Didn't check if this is feasible.

Reply:
I'm pretty sure engine-io will handle the serialization for us, so shouldn't need to handle the JSON encoding manually